packages/catalog-process/src/services/catalogService.ts

```diff
@@ -625,6 +668,28 @@ export function catalogServiceBuilder(
         })
       );
     },
+    async createRiskAnalysis(
+      eserviceId: EServiceId,
+      eserviceRiskAnalysisSeed: EServiceRiskAnalysisSeed,
+      authData: AuthData
+    ): Promise<string> {
+      logger.info(`Creating Risk Analysis for EService ${eserviceId}`);
+
+      const eservice = await readModelService.getEServiceById(eserviceId);
+      const tenant = await readModelService.getTenantById(
+        authData.organizationId
+      );
+
+      return await repository.createEvent(
+        createRiskAnalysisLogic({
+          eserviceId,
+          eserviceRiskAnalysisSeed,
+          authData,
+          eservice,
+          tenant,
+        })
+      );
+    },
```

Comment:
Could you refactor this function according to the updated pattern? You can take the others as a reference, after updating the branch.

Reply:
Done! Thanks :)